src/pe/delay_import.rs

```diff
@@ -0,0 +1,529 @@
+use alloc::vec::Vec;
+use scroll::ctx;
+use scroll::{Pread, Pwrite, SizeWith};
+
+use crate::container::{Container, Ctx};
+use crate::error;
+use crate::pe::data_directories;
+use crate::pe::import::{IMPORT_BY_ORDINAL_32, IMPORT_BY_ORDINAL_64};
+use crate::pe::options;
+use crate::pe::section_table;
+use crate::pe::utils;
+
+/// A binary parsing context for PE parser
+#[derive(Debug, Copy, Clone, PartialEq)]
+pub struct PeCtx<'a> {
+    pub container: Container,
+    pub le: scroll::Endian,
+    pub sections: &'a [section_table::SectionTable],
+    pub file_alignment: u32,
+    pub opts: options::ParseOptions,
+    pub bytes: &'a [u8], // full binary view
+}
```

Comment:
if this is meant to be a generic Ctx for parsing PE binaries, then it shouldn't be in this module, for example. a better place is `mod` or even better a `ctx.rs` file in the `src/pe` directory.